established database connection

FelixLetheren opened the pull request.

FelixLetheren committed:
established database connection

AlisonYoon commented:
looks good

sjr-mayden commented on php-files/index.php:
```diff
@@ -0,0 +1,14 @@
+
+$db -> setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$sql = $db->prepare('SELECT * FROM `fish`;');
```
SELECTing * is bad practice as it makes the query very slow, it's better to be specific as to which fields you want to SELECT

sjr-mayden commented on php-files/index.php:
```diff
@@ -0,0 +1,14 @@
+
+$db = new PDO('mysql:host=192.168.20.20; dbname=fish-finder', 'root', '');
+
+$db -> setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```
Extra spacing about the object accessor

sjr-mayden commented on php-files/index.php:
```diff
@@ -0,0 +1,14 @@
+
+$sql = $db->prepare('SELECT * FROM `fish`;');
+
+$sql ->execute();
```
Extra spacing about the object accessor

sjr-mayden commented on php-files/index.php:
```diff
@@ -0,0 +1,14 @@
+
+$sql ->execute();
+
+$collection = $sql -> fetchAll();
```
Extra spacing about the object accessor

sjr-mayden commented on php-files/index.php:
```diff
@@ -0,0 +1,14 @@
+<?php
```
Why isn't in the root of the project, it should be

FelixLetheren committed:
Formatting changes
Removed whitespace from object selectors.
changed SQL statement to be more specific.
Moved index.php into folder root instead of seperate file
